pkgs/applications/misc/ecm-tools/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchgit }:
+stdenv.mkDerivation rec {
+  name = "ecm-tools";
+  version = "master";
```

Comment:
hi the current maintainer doesn't have any version number. should i just go for 1.0?